torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -487,7 +487,7 @@ def forward(self, *inputs: Any) -> Any:
         if isinstance(orig_image, torch.Tensor):
             image = orig_image
         else:  # isinstance(inpt, PIL.Image.Image):
-            image = F.to_image_tensor(orig_image)
+            image = F.pil_to_tensor(orig_image)
```

Comment:
Perf: There is no reason to wrap the PIL into an Image object because few lines below the meta-data will be lost due to the `view()` call.